frontend/src/concepts/areas/types.ts

```diff
@@ -89,23 +92,6 @@ export enum SupportedArea {
 }
 
 export type SupportedAreaType = SupportedArea | string;
-/** Components deployed by the Operator. Part of the DSC Status. */
-export enum StackComponent {
```

Comment:
A lot of changes could have been avoided by repurposing the values of this enum to the new one and removing `DataScienceStackComponent`.

Reply:
I understand, which is why i suggested deleting `DataScienceStackComponent` and updating `StackComponent` with the new values. The net result being a single enum.